.github/workflows/lasuite.yaml

```diff
@@ -0,0 +1,51 @@
+name: lasuite
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  lint-git:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' # Makes sense only for pull requests
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
i'm really not a user of github actions, just asking here: any reason to specificly use v2 ?

Reply:
No, just being aligned with other project CI, and I am not even sure that's a good reason. People CI might be outdated too. As it's not a big deal, I'll just ignore this for now and discuss it with @rouja !